apps/spruce/src/pages/adminSettings/tabs/GeneralTab/ProvidersTab/getFormSchema.ts

```diff
@@ -0,0 +1,62 @@
+import { GetFormSchema } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import { fullWidthCss, objectGridCss } from "../../sharedStyles";
+import { containerPools, aws, repoExceptions } from "./schemaFields";
+
+export const getFormSchema = (): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as const,
+    properties: {
+      providers: {
+        type: "object" as const,
+        title: "",
+        properties: {
+          containerPools: {
+            type: "object" as const,
+            title: "Container Pools",
+            properties: {
+              ...containerPools.schema,
+            },
```

Comment:
[nit] No spread here and below
```suggestion
            properties: containerPools.schema,
```